### ISSUE TITLE
Lot

### ISSUE THREAD

**Issue by ViTulyankin** [Ladysnake/Effective#232]

**Comment by Pyrofab** [comment]
Fixed the first one, not familiar enough with cascade sound code to fix the second one, rest may have to wait for a Lodestone update

**Comment by CarlMichaelFabe** [comment]
Tried [736433b](https://github.com/Ladysnake/Effective/commit/736433b096dcd561d70a5f737a787a3c7e891d0c) and now the splashes are spawning downwards underwater.

https://github.com/Ladysnake/Effective/assets/116660522/48523314-c220-49a0-be53-949b573934e0

**Comment by szunakabeats** [comment]
yea the cascade sounds are bugged. they are very quiet and for some reason come from the wrong direction.
maybe it's not compatible with the ambientsounds mod ?

**Comment by ViTulyankin** [comment]
> rest may have to wait for a Lodestone update

Well, it seems ike Lodestone was ported to 1.20.4 Fabric :)

**Comment by cph101** [comment]
Yeah, lodestone isn’t compatible with iris shaders (It modifies core particle shaders and Iris being the dumbass it is just completely overrides them). I am going to find a fix to the splash particles one war or another. It may be possible to just rotate it 180 degrees across either the x or the z axis, just putting that out there if anyone can try before I get round to it.

**Comment by doctor4t** [comment]
All but the last one fixed in the latest Fabric version
